CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 ## 1.4.0 (04-28-2017)
 * [Breaking change] [getThreadHistory](/DOCS.md#getThreadHistory): update parameters - no more start & end params; replaced with amount
-* [New] [setMessageReaction](/DOCS.md#setMessageReaction) - set reaction on message
-* [New] [forwardAttachment](/DOCS.md#forwardAttachment) - send attachment to array of users
-* [setMessage](/DOCS.md#apisendmessagemessage-threadid-callback): added Mentions field
+* [New] [setMessageReaction](/DOCS.md#setMessageReaction): set reaction on message
+* [New] [forwardAttachment](/DOCS.md#forwardAttachment): send attachment to array of users
+* [setMessage](/DOCS.md#sendMessage): added Mentions field
 * [sendTypingIndicator](/DOCS.md#sendTypingIndicator): make callback optional
+* [getUserID](/DOCS.md#getUserID): support searches for more than just names, and add a type parameter
```

Comment:
I would write "now returns all results instead of just users" because it was throwing error before
And as far as I remember "type" is not a parameter but a field in result object (?)